packages/lib/shim-init-node.js

```diff
@@ -490,6 +490,10 @@ function shimInit(options = null) {
 			method: method,
 			path: url.pathname + (url.query ? `?${url.query}` : ''),
 			headers: headers,
+			timeout: options.timeout,
+			// 21 redirects is the default amount from follow-redirects library
+			// 20 seems to be the max amount that most popular browsers will allow
+			maxRedirects: options.maxRedirects ?? 21,
```

Comment:
Could you give timeout and maxRedirects default values at the top of the function (like it's done for the "method" option). In more recent code we generally have the pattern `options = { /* Default values */, ...options }` so that we know what the default values are going to be and we also know they will actually be defined throughout the function

Reply:
21 redirects seem massive. Is there any documentation somewhere as to why that might be needed?

Reply:
The only reason I could find is that Chrome/Firefox seems to accept 20 redirects before failing, but I could try to look a little harder.

Reply:
Ok that makes sense for a browser which would have a lot more cases to deal with than us. Let's leave it as 21 by default for now anyway since we can now customise it if needed.